AzureMonitorAgent/ama_tst/modules/connect/connect.py

```diff
@@ -8,6 +8,11 @@
 from .check_endpts     import check_internet_connect, check_ama_endpts
 from .check_imds       import check_imds_api
 
+try:
+    FileNotFoundError
+except NameError:
+    FileNotFoundError = IOError
```

Comment:
I think this might need to be OSError instead of IOError - for the troubleshooter, we want to keep the output as clean as possible

Reply:
Checked and it is not only the python patch differences, but even within the same version FileNotFound error is thrown from different paths in TSA, some methods throw it from file I/O operations (see helpers.py) and some are list dir ops like this example stack I attached to the PR. Only in list dir it is OSError but in  file operations it is I/O error. So leaving it as is.